chrome-version/query-lens.test.js

```
@@ -0,0 +1,137 @@
+const { JSDOM } = require('jsdom');
+const fs = require('fs');
+const path = require('path');
+
+// Load the actual QueryLens implementation
+const queryLensCode = fs.readFileSync(path.join(__dirname, 'query-lens.js'), 'utf8');
+global.QueryLens = eval(`(function() { ${queryLensCode}; return QueryLens; })()`);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)